src/Symfony/Component/Validator/Constraints/AbstractComparisonValidator.php

```diff
@@ -63,6 +83,18 @@ public function validate($value, Constraint $constraint)
         }
     }
 
+    private function getPropertyAccessor()
+    {
+        if (null === $this->propertyAccessor) {
+            if (!class_exists(PropertyAccess::class)) {
```

Comment:
What about failing earlier instead when the constraint definition is evaluated (i.e. in the constructor of `AbstractComparison`)?